subworkflows/ebi-metagenomics/taxonomy/tests/main.nf.test

```diff
@@ -0,0 +1,29 @@
+nextflow_workflow {
+
+    name "Test subworkflows/taxonomy/tests/main.nf.test"
+    script "../main.nf"
+    workflow "TAXONOMY"
+
+    test("Run the pipeline on the test dataset") {
+        when {
+            params {
+                output_path    = "${outputDir}"
+            }
+            workflow {
+                """
+                input[0] = [ [id:'test'], file("${moduleDir}/tests/data/contigs.fasta", checkIfExists: true) ]
+                input[1] = [ [id:'test'], file("${moduleDir}/tests/data/proteins.fasta", checkIfExists: true) ]
+                input[2] = [ [id:'test'], file("${moduleDir}/tests/data/db", checkIfExists: true) ]
```

Comment:
This is where you should change your unit test after you changed how it creates the channel for the DB